docs/extend/modular-resources/configure.md

```diff
@@ -1,30 +1,171 @@
 ---
-title: "Configure your module and modular resource"
+title: "Configure a module on your robot"
 linkTitle: "Configure"
 weight: 30
 type: "docs"
 tags: ["server", "rdk", "extending viam", "modular resources", "components", "services"]
-description: "Use the Viam module system to implement custom resources that can be included in any Viam-powered robot."
+description: "Configure a module on your robot."
 no_list: true
 ---
 
-Configure a module on your robot to make one or more modular resources available for configuration.
+Add and configure a module on your robot to make one or more modular resources available for configuration.
```

Comment:
```suggestion
Add a module and configure a modular resource to add extended functionality to your robot.
```
or perhaps
```suggestion
To add a modular resource to your robot, first configure a module and then configure the modular component or service.
```
or maybe delete this sentence entirely? It's a bit tautological I think.
Also isn't "add and configure" redundant? You have to configure the module in the `modules` section of your config, and configure the resource in the `components` or `services` part, right? But you could call both of those things "adding" or "configuring"

Reply:
Addressed also in https://github.com/viamrobotics/docs/pull/1705.